src/screens/Customers/components/MeterInfoContainer.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import { createStyles, withStyles, Theme } from '@material-ui/core/styles';
+import { Typography } from '@material-ui/core';
+import { CardPropsInfo } from '../../../components/OutlinedCardList';
+import IconButton from '@material-ui/core/IconButton';
+import { Create as CreateIcon } from '@material-ui/icons';
+
+const styles = (theme: Theme) =>
+  createStyles({
+    root: {
+      width: '50%',
+      display: 'flex',
+    },
+    info: {
+      width: '100%',
+      marginBottom: '8px',
+      padding: '0px 12px 4px',
+      border: '1px solid',
+      height: '60px',
+      maxHeight: '60px',
+      borderRadius: '6px',
+    },
+    icon: {
+      fontSize: '14px',
+      marginLeft: '5px',
+      marginTop: '-4px',
+      color: theme.palette.primary.main,
+    },
+    inline: {
+      display: 'inline-flex',
+    },
+  });
+
+interface MeterInfoContainerProps {
+  classes: { root: string; info: string; icon: string; inline: string; };
+  floatRight: boolean;
+  info: CardPropsInfo;
+  editable?: boolean;
+  gray?: boolean;
```

Comment:
It seems like this is only ever either editable or gray, so could you eliminate the `gray` prop?

Reply:
this is also a case of if you look at smart meter vs no meter, there's not always gray text AND gray background